transactions/switchboard/setup_royalty_account.cdc

```diff
@@ -4,87 +4,68 @@ import "FlowToken"
-        if !signer.getCapability<&{FungibleToken.Receiver}>
-                                           (FiatToken.VaultReceiverPubPath).check() {
-            // if it's not, create a public capability to the USDC vault exposing 
-            // the deposit function through the {FungibleToken.Receiver} interface
-            signer.unlink(FiatToken.VaultReceiverPubPath)
-            signer.link<&FiatToken.Vault>(FiatToken.VaultReceiverPubPath, 
-                                                  target: FiatToken.VaultStoragePath)
+        if !signer.capabilities.get<&{FungibleToken.Receiver}>
+                                           (FiatToken.VaultReceiverPubPath)!.check() {
+            // if it's not, create a public capability to the USDC vault
+            let tokenCap = signer.capabilities.storage.issue<&FiatToken.Vault>(FiatToken.VaultStoragePath)
+            signer.capabilities.publish(tokenCap, at: FiatToken.VaultReceiverPubPath)
+            let receiverCap = signer.capabilities.storage.issue<&FiatToken.Vault>(FiatToken.VaultStoragePath)
+            signer.capabilities.publish(receiverCap, at: FiatToken.VaultBalancePubPath)
         }
-        self.fiatTokenVaultCapability = signer.getCapability<&{FungibleToken.Receiver}>(FiatToken.VaultReceiverPubPath)
+        self.fiatTokenVaultCapability = signer.capabilities.get<&{FungibleToken.Receiver}>(FiatToken.VaultReceiverPubPath)
         
         // Check if the account already has a Switchboard resource
-        if signer.borrow<&FungibleTokenSwitchboard.Switchboard>
+        if signer.storage.borrow<&FungibleTokenSwitchboard.Switchboard>
                                 (from: FungibleTokenSwitchboard.StoragePath) == nil {
             // If not, create a new Switchboard resource and put it into storage
-            signer.save(<- FungibleTokenSwitchboard.createSwitchboard(), 
+            signer.storage.save(<- FungibleTokenSwitchboard.createSwitchboard(), 
                                             to: FungibleTokenSwitchboard.StoragePath)
         }
         // Check if the receiver capability is linked on the receiver path
-        if !signer.getCapability
+        if !signer.capabilities.get
                       <&FungibleTokenSwitchboard.Switchboard>
-                              (FungibleTokenSwitchboard.ReceiverPublicPath).check() {
-            // if it's not, create a public capability to the Switchboard exposing 
-            // the deposit function through the {FungibleToken.Receiver} interface
-            signer.unlink(FungibleTokenSwitchboard.ReceiverPublicPath)
-            signer.link<&FungibleTokenSwitchboard.Switchboard>(
-                                         FungibleTokenSwitchboard.ReceiverPublicPath,
-                                        target: FungibleTokenSwitchboard.StoragePath)
+                              (FungibleTokenSwitchboard.ReceiverPublicPath)!.check() {
```

Comment:
Same question as above. Applies to at least one more check below

Reply:
yep, I'll keep that in mind when I get to testing these after `FiatToken` is upgraded